kubejs/assets/kubejs/lang/en_us.json

```diff
@@ -186,14 +186,14 @@
     "fluid.kubejs.molten_aeternium": "Molten Aeternium",
     "fluid.kubejs.molten_alfsteel": "Molten Alfsteel",
     "fluid.kubejs.molten_gaia_spirit": "Molten Gaia Spirit",
-    "fluid.kubejs.molten_niotic_crystal": "Molten Niotic Crystal",
-    "fluid.kubejs.molten_nitro_crystal": "Molten Nitro crystal",
-    "fluid.kubejs.molten_spirited_crystal": "Molten Spirited Crystal",
+    "fluid.kubejs.molten_niotic": "Molten Niotic Crystal",
+    "fluid.kubejs.molten_nitro": "Molten Nitro crystal",

```

Comment:
Crystal needs a capital C, somehow missed it in my original pr 😅

Reply:
Oh jeez I was trying to find that! Thanks